md/languages.md

```diff
@@ -1,70 +1,62 @@
+3. Select a new language from the drop-down list
 
-1. Log in to [Crowdin](http://translate.bonitasoft.org/) and download the archive file `bonita-bpm-60.zip`.
-2. Unzip the `bonita-bpm-60.zip` file to a folder, and go to the `/master` folder. Find the following files:
-   * `/bonita-web/portal/portal_ru_RU.po`
-   * `/bonita-web/portal/portal-js_ru_RU.po`
-   * `/bonita-web-sp/portal/mobile_ru_RU.po`
-   * `/bonita-web-sp/portal/portal-js-sp_ru_RU.po`
-   * `/bonita-web-sp/portal/portal-sp_ru_RU.po`
-3. Copying these files to the Portal localization folder, `[TOMCAT_INSTALL_FOLDER]/my_custom_i18n/`
-4. Edit `[TOMCAT_INSTALL_FOLDER]/bin/setEnv.sh` (under Unix) or `[TOMCAT_INSTALL_FOLDER]\bin\setEnv.bat` (under Windows) and append the modify the existing line starting with **CATALINA_OPTS=...** :
+## Add a language to Bonita BPM Portal
 
-   CATALINA_OPTS=-Dorg.bonitasoft.i18n.folder=${CATALINA_HOME}/my_custom_i18n
-5. Restart Tomcat server 
+::: warning
+We are currently updating the translation process. Among other things this update impacts the Crowdin project structure. Instruction below might be temporarily inaccurate.
+:::
 
-## How to translate Bonita BPM Portal to any other language
+::: info
+Instructions below assume that the language you want to add is already available in the community translation project. If translation is not available see [Translate Bonita BPM Portal](#Translate-Bonita-BPM-Portal) for instruction about how to collaborate to a new or ongoing translation.
+:::
 
-1. Go to [crowdin.com](https://crowdin.com).
-2. Create an account and join the project **Bonita BPM 6.0**. The same project is used for 6.x and 7.x translations.
-3. Once you have joined the [project](https://crowdin.com/project/bonita-bpm-60), there are 2 possible cases:
+Instructions below explain how to add support for a language to Bonita BPM Portal. Steps below involve using content created by Bonitasoft community and therefor not officially supported.
 
-#### Complete an ongoing translation
+1. Go to [Bonita BPM translation project](http://translate.bonitasoft.org/)
+1. Click the **_Download_** icon to download the zip archive file containing the translations into all the available languages. The archive is called `bonita-bpm-60.zip` even for Bonita BPM 7 versions.
+1. Go to the `master/bonita-web/portal` folder.
+1. Find and extract the `.po` files for the language you want to add (several files for a single language). Each `.po` file has a language indicator and a locale indicator. For example, the files for the Brazilian Portuguese language end with `pt_BR.po`.
+1. For Subscription editions, you also need to get the files from `master/bonita-web-sp/portal` folder.
+1. Copy the `.po` files to a new folder of your choice, preferably in a sub-folder inside your bundle. E.g. `[TOMCAT_INSTALL_FOLDER]/i18n/`. For Bonita BPM Studio, Tomcat installation folder is located in `/workspace/tomcat`.
```

Comment:
be careful, `workspace` is only the default location. If a user has changed it subscription version, the folder may not be `workspace`....